client/src/components/Ticketing/ticketingmanager/Events/Add_event/CreateEventPage.tsx

```diff
@@ -51,15 +52,15 @@ const CreateEventPage = () => {
         'Authorization': `Bearer ${token}`,
       },
       method: 'POST',
-      body: JSON.stringify({eventName, eventDesc, imageUrl}),
+      body: JSON.stringify({seasonId, eventName, eventDesc, imageUrl}),
     });
 
     if (createPlayRes.ok) {
       const eventData = await createPlayRes.json();
-      console.log(eventData);
-      const {id} = eventData.data[0];
+      const id = eventData.data[0].eventid;
+      console.log(id);
       const showingdata = showings.map(formatShowingData(id));
-
+      console.log(showingdata);
```

Comment:
All the console logs, while good for debugging, make the console really spammy so unless they are important to have in deployment we should remove them